internal/osutil/homedir.go

```diff
@@ -0,0 +1,13 @@
+package osutil
+
+import "os"
+
+// UserHomeDir is similar to os.UserHomeDir, but prefers $HOME when available
+// over other options (such as USERPROFILE on Windows).
+func UserHomeDir() (string, error) {
+	h := os.Getenv("HOME")
+	if h != "" {
```

Comment:
Is there any preference of this style over?:
```go
	if h := os.Getenv("HOME"); h != "" {
```

Reply:
No, in fact I think we _should_ prefer it that way. Just hammered out the code quickly.